### ISSUE TITLE
Add the app to f-froid repo

### ISSUE THREAD

**Issue by not-programmer98** [BeamlakAschalew/cinemax#60]
### Steps to reproduce

...

### Expected behavior

...

### Actual behavior

...

### Android version

.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by BeamlakAschalew** [comment]
Sorry for the late reply, I'll look into it

**Comment by DarkCrypt** [comment]
F-Droid will probably deny this app because of Google Services.

**Comment by BeamlakAschalew** [comment]
> F-Droid will probably deny this app because of Google Services.

so nothing Google related including Firebase isn't allowed?

**Comment by DarkCrypt** [comment]
> so nothing Google related including Firebase isn't allowed?

Correct, the entire app has to be fully FLOSS. No proprietary software allowed.
